src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -476,6 +561,16 @@ CHIP_ERROR MinMdnsResolver::BrowseNodes(DiscoveryType type, DiscoveryFilter filt
             qname = CheckAndAllocateQName(subtypeStr, "_sub", "_chipc", "_udp", "local");
         }
         break;
+    case DiscoveryType::kCommissionerNode:
+        if (filter.type == DiscoveryFilterType::kNone)
+        {
+            qname = CheckAndAllocateQName("_chipd", "_udp", "local");
```

Comment:
Once https://github.com/project-chip/connectedhomeip/pull/7116 goes in, this will have to be changed. Not sure who will get there first.